services/cd-service/pkg/service/git.go

```diff
@@ -56,35 +56,6 @@ func (s *GitServer) GetGitTags(ctx context.Context, _ *api.GetGitTagsRequest) (*
 	return &api.GetGitTagsResponse{TagData: tags}, nil
 }
 
-func getProductSummaryForEnv(ctx context.Context, transaction *sql.Tx, state *repository.State, overviewService *OverviewServiceServer, envName string, appDetailsCache map[string]*api.GetAppDetailsResponse) ([]api.ProductSummary, error) {
```

Comment:
This was implemented via the GetAppDetails endpoint. Adapting the GetProductSummary to go back in time meant that we would have to also adapt the GetAppDetails endpoint to do so, which might have been interesting, but falls out of scope of Ref: SRX-S3J08Q. It also returns a lot of information that the GetProductSummary simply does not need.